### DISCUSSION TITLE
各位前辈们的绳网等级都是怎么升的呀

### DISCUSSION THREAD

**Discussion by yuxigongzi** [share121/inter-knot#2747] [其他]
我虽然注册绳网挺早的，但我前四年做的委托都是移植搬运融合调试的旧艾丽都的工程师的项目，没有上传绳网，所以没有得到绳网经验，到现在等级都还是0级；现在的委托又涉及保密，估摸着还会保持0级很久？
前辈们大佬们都是做的什么委托提升的绳网等级啊？

**Comment by lllliufeng** [comment]
维修HDD（确信

**Comment by CBBC123** [comment]
不知道，莫名其妙就一级了

**Comment by LittelSpecial** [comment]
楼上说的对✓😙

**Comment by shouyeren7** [comment]
啊？问下作者看看?

**Comment by zhixuan45** [comment]
emmm我做的基本上是插件工程的项目，不过提交次数比较多就11了

**Comment by jsishsjsj** [comment]
有多少个仓库就多少级<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by zhixuan45** [reply]
我13仓库但是只有11级///

**Reply by wtnl551** [reply]
是公开仓库数量

**Reply by zhixuan45** [reply]
> 是公开仓库数量

![image](https://github.com/user-attachments/assets/5b78733e-24cb-4927-95db-3c8872429c07)
12-1？

**Reply by isle200inf** [reply]
![image](https://github.com/user-attachments/assets/468fa533-2597-4629-b52b-02879e714294)
11

**Comment by bangbu01** [comment]
多发点帖子就好了
